### DISCUSSION TITLE
Remove Icon button

### DISCUSSION THREAD

**Discussion by matteo-dinoia** [FossifyOrg/General-Discussion#632] [Ideas]
Add option to remove the icon button and increase the space bar lenght to fit. 
I would like this because i'm used to a larger space bar and i never use this button anyway.
Maybe the two feature (icon and language) could be moved to the comma/dot long press action when this option is enabled.

**Comment by Aga-C** [comment]
Please create a new [feature request](https://github.com/FossifyOrg/Keyboard/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for this.
